sgx/sync/src/mutex.rs

```diff
@@ -356,7 +356,7 @@ impl<T: ?Sized> SgxMutex<T> {
     /// the current thread.
     pub fn lock(&self) -> LockResult<SgxMutexGuard<T>> {
         unsafe {
-            drop(self.inner.lock());
+            let _ = self.inner.lock();
```

Comment:
I don't know why these were `drop()` before. The return type is a `Result<(), error_code>` where the error code is an integer.